XPlot.DotNet.Interactive.KernelExtensions/PlotlyChartExtensions.cs

```diff
@@ -1,32 +1,28 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
 using System.Text;
-using HtmlAgilityPack;
+using Microsoft.DotNet.Interactive.Formatting;
 using XPlot.Plotly;
 
 namespace XPlot.DotNet.Interactive.KernelExtensions
 {
     public static class PlotlyChartExtensions
     {
-        public static string GetChartHtml(this PlotlyChart chart)
+        public static string GetHtml(this PlotlyChart chart)
         {
-            var document = new HtmlDocument();
-            document.LoadHtml(chart.GetInlineHtml());
-
-            var divNode = document.DocumentNode.SelectSingleNode("//div");
-            var scriptNode = document.DocumentNode.SelectSingleNode("//script");
-
-            var newHtmlDocument = new HtmlDocument();
-
-            newHtmlDocument.DocumentNode.ChildNodes.Add(divNode);
-            newHtmlDocument.DocumentNode.ChildNodes.Add(GetScriptNodeWithRequire(scriptNode));
-
-            return newHtmlDocument.DocumentNode.WriteContentTo();
+            dynamic _ = new PocketView();
+            var div = _.div[style: $"width: {chart.Width}px; height: {chart.Height}px;", id: chart.Id]();
+            var js = chart.GetInlineJS().Replace("<script>", string.Empty).Replace("</script>",string.Empty).Trim();
+            
+            return $@"{div}
```

Comment:
You don't need to use an interpolated string here, and in fact you're more likely to end up with HTML encoding issues if you do this.

This is the preferred approach:

```csharp
return div(GetScriptNodeWithRequire(js));
```

`GetScriptNodeWithRequire` should return `IHtmlContent` since it does not need to be encoded.

Reply:
that will surrounds the div with another div. we were returning a different shape before